src/Dockerfile

```diff
@@ -9,11 +9,17 @@ ENV ASPNETCORE_HTTP_PORTS=8080
 
 WORKDIR /app
 COPY services/EdFi.DataManagementService.Api/*.csproj ./services/EdFi.DataManagementService.Api/
+COPY backend/EdFi.DataManagementService.Backend/*.csproj ./backend/EdFi.DataManagementService.Backend/
+COPY backend/EdFi.DataManagementService.Backend.Mssql/*.csproj ./backend/EdFi.DataManagementService.Backend.Mssql/
+COPY backend/EdFi.DataManagementService.Backend.Postgresql/*.csproj ./backend/EdFi.DataManagementService.Backend.Postgresql/
```

Comment:
I wonder if `COPY **/*.csproj` would work?

Reply:
I couldn't find a way to use wildcards to get these COPY instructions to work right.

Reply:
Thanks for trying. I suspected it wouldn't work out, though I hoped I was just overlooking something in my own reading.